ui/app/utils/tensorzero/tensorzero.ts

```diff
@@ -1792,6 +1801,130 @@ export class TensorZeroClient {
     return (await response.json()) as GetEvaluationResultsResponse;
   }
 
+  // =========================================================================
+  // Autopilot API Methods
+  // =========================================================================
+
+  /**
+   * Lists autopilot sessions with optional pagination.
+   */
```

Comment:
I think we should have a separate class for autopilot stuff, one because this file is already too big and we're dealing with a lot of merge conflicts here, and two I think autopilot behavior will likely be different from regular gateway requests down the line. I doubt you need both autopilot and regular gateway methods in the same UI views so it shouldn't introduce additional complexity.